src/automation/atm_dicts.py

```diff
@@ -0,0 +1,444 @@
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+#30
+
+ssmi_ssmit = InventoryInfo(
+    obs_name='ssmi_ssmit',
+    key='observations/reanalysis/ssmi/ssmit/%Y/%m/bufr/gdas.%Y%m%d.t%Hz.ssmit.tm00.bufr_d',
+    start='20000316T120000Z',
+    s3_prefix='observations/reanalysis/ssmi/ssmit/%Y/%m/bufr/',
+    bufr_files='gdas.%z.ssmit.tm00.bufr_d',
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+ssmis_eumetsat = InventoryInfo(
+    obs_name='ssmis_eumetsat',
+    key='observations/reanalysis/ssmis/eumetsat/%Y/%m/bufr/gdas.%Y%m%d.t%Hz.ssmisu.tm00.bufr_d',
+    start='20051101T000000Z',
+    s3_prefix='observations/reanalysis/ssmis/eumetsat/%Y/%m/bufr/',
+    bufr_files='gdas.%z.ssmisu.tm00.bufr_d',
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+ssmis_ssmisu = InventoryInfo(
+    obs_name='ssmis_ssmisu',
+    key='observations/reanalysis/ssmis/ssmisu/%Y/%m/bufr/gdas.%Y%m%d.t%Hz.ssmisu.tm00.bufr_d',
+    start='20091001T000000Z',
+    s3_prefix='observations/reanalysis/ssmis/ssmisu/%Y/%m/bufr/',
+    bufr_files='gdas.%z.ssmisu.tm00.bufr_d',
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+ssu_1bssu = InventoryInfo(
+    obs_name='ssu_1bssu', 
+    key='observations/reanalysis/ssu/1bssu/%Y/%m/bufr/gdas.%Y%m%d.t%Hz.1bssu.tm00.bufr_d',
+    start='19790101T000000Z',
+    s3_prefix='observations/reanalysis/ssu/1bssu/%Y/%m/bufr/',
+    bufr_files='gdas.%z.1bssu.tm00.bufr_d',
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+trmm_nasa_tmi = InventoryInfo(
+    obs_name='trmm_nasa_tmi',
+    key='observations/reanalysis/trmm/nasa/tmi/%Y/%m/bufr/tmi.V05A.%Y%m%d.t%Hz.bufr',
+    start='19980101T000000Z',
+    s3_prefix='observations/reanalysis/trmm/nasa/tmi/%Y/%m/bufr/',
+    bufr_files='tmi.V05A.%z.bufr',
+    nceplibs_cmd=au.NCEPLIBS_SINV,
+    platform=au.CLEAN_PLATFORM,
+    cycling_interval=au.CYCLING_6H,
+    s3_bucket=au.REANALYSIS_BUCKET,
+)
+
+#35
+
+atm_infos = [airs_airsev, airs_aqua, amsua_1bamua, amsua_nasa_aqua, amsua_nasa_r21c, amsub_1bamub, amv_merged, atms_atms,
+             avhrr_avcsam, avhrr_avcspm, conv_prepbufr_acft_profiles, conv_prepbufr, cris_cris, cris_crisf4, geo_geoimr,
+             geo_goesfv, geo_goesnd, gmi_nasa_gmiv7, gps_eumetsat, gps_gpsro, hirs_1bhrs2, hirs_1bhrs3, hirs_1bhrs4,
+             iasi_mtiasi, mhs_1bmhs, msu_1bmsu, ozone_sbuv_v87, saphir_saphir, seviri_sevcsr, ssmi_eumetsat,
+             ssmi_ssmit, ssmis_eumetsat, ssmis_ssmisu, ssu_1bssu, trmm_nasa_tmi]
+
+rerun = [atms_atms, cris_crisf4]
```

Comment:
we need to support custom cases in the future PRs with more flexibility so it doesn't require source code changes.

Reply:
added an issue for this to be addressed in the future